src/libs/ReportActionsUtils.ts

```diff
@@ -1037,8 +1037,8 @@ function getAllReportActions(reportID: string): ReportActions {
 function isReportActionAttachment(reportAction: OnyxInputOrEntry<ReportAction>): boolean {
     const message = getReportActionMessage(reportAction);
 
-    if (reportAction && ('isAttachment' in reportAction || 'attachmentInfo' in reportAction)) {
-        return reportAction?.isAttachment ?? !!reportAction?.attachmentInfo ?? false;
+    if (reportAction && ('isAttachmentOnly' in reportAction || 'isAttachmentWithText' in reportAction)) {
+        return reportAction?.isAttachmentOnly ?? reportAction?.isAttachmentWithText ?? false;
```

Comment:
reportAction can't be null
```suggestion
        return reportAction.isAttachmentOnly ?? reportAction.isAttachmentWithText ?? false;
```